python/ray/train/session.py

```diff
@@ -252,6 +255,18 @@ def _warn_session_misuse(fn_name: str):
         )
 
 
+def _raise_session_misuse(fn_name: str):
+    """Raises RuntimeError on provided fn being used outside of session.
+
+    Args:
+        fn_name (str): The name of the function used outside of session.
+    """
+    raise RuntimeError(
+        f"`train.{fn_name}()` is meant to only be called inside a training function "
```

Comment:
Ah I see what you mean now- `get_accelerator` and `set_accelerator` are not exposed to the user so then this error message doesn't really make sense. 

We can change this to a more generic `prepare/accelerate utility functions are meant to only be called`..., and we can change the function name to `_raise_accelerator_session_misuse` so that it's only used for accelerators.